agent/apps/ai_guardrails/app_code/cores/orchestrator.py

```
@@ -0,0 +1,125 @@
+    def evaluate(self, content: str, user, direction: str = 'both', request=None) -> Dict[str, Any]:
+        """
+        ارزیابی محتوای ورودی/خروجی و بازگرداندن نتیجه تصمیم‌گیری
+        """
+        active_policies = GuardrailPolicy.objects.filter(is_active=True).order_by('priority')
+        direction = direction if direction in {"input", "output", "both"} else "both"
+        if direction in ['input', 'output']:
+            active_policies = active_policies.filter(Q(applies_to=direction) | Q(applies_to='both'))
+        matches: list[dict[str, Any]] = []
+        reasons: list[str] = []
+        max_severity = 0
+        first_matched_rule = None  # RedFlagRule | None
```

Ruff (0.12.2)

Local variable `first_matched_rule` is assigned to but never used

Remove assignment to unused variable `first_matched_rule`

(F841)